### PR TITLE
Ledger Card

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-425

## What changed (plus any additional context for devs)
- created Ledger card
- refactored/cleaned up `GenericCard` and other existing cards

## Screen recordings / screenshots
https://www.loom.com/share/714c50adf5614e00ba8acd4eef437ddf

also tested iPhone SE, looks good

### CODE REVIEW

File: src/components/cards/OpRewardsCard.tsx [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
+} from '@/design-system';
 import { ButtonPressAnimation } from '@/components/animations';
-import { Image, StyleSheet } from 'react-native';
+import { Image } from 'react-native';
 import OpRewardsCardBackgroundImage from '../../assets/opRewardsCardBackgroundImage.png';
-import { useTheme } from '@/theme';
 import * as i18n from '@/languages';
 import { useNavigation } from '@/navigation';
 import Routes from '@/navigation/routesNames';
+import { colors } from '@/styles';
```
Comment (benisgold) [inline src/components/cards/OpRewardsCard.tsx:17]: @jkadamczyk this was the color fix I DM'ed you about

Comment (jkadamczyk) [reply src/components/cards/OpRewardsCard.tsx:17]: Thank you!

File: src/screens/discover/components/DiscoverHome.js [hunk 2]
```diff
@@ -20,12 +21,14 @@ import {
...
 } from '@/components/cards/utils/constants';
 import { OpRewardsCard } from '@/components/cards/OpRewardsCard';
+import { LedgerCard } from '@/components/cards/LedgerCard';
 
 export default function DiscoverHome() {
   const { network } = useAccountSettings();
   const accountAsset = useAccountAsset(ETH_ADDRESS);
   const profilesEnabled = useExperimentalFlag(PROFILES);
   const optimismRewardsEnabled = useExperimentalFlag(OP_REWARDS);
+  const hardwareWalletsEnabled = useExperimentalFlag(HARDWARE_WALLETS);
```
Comment (jkadamczyk) [inline src/screens/discover/components/DiscoverHome.js:31]: keep in mind useExperimentalFlag doesn't work in TestFlight, check my change here https://github.com/rainbow-me/rainbow/blob/f52a43af504c97810e155aa179d0a539fc40c0f0/src/config/experimentalHooks.ts#L8

**Review by jkadamczyk — APPROVED** [review]
Looks good to me!

File: src/components/cards/ReceiveAssetsCard.tsx [hunk 3]
```diff
@@ -42,12 +42,12 @@ export const ReceiveAssetsCard = () => {
     [accountAddress, isToastActive, setClipboard, setToastActive]
   );
 
-  const onPressQRCode = useCallback(() => {
+  const onPressQRCode = () => {
```
Comment (estrattonbailey) [inline src/components/cards/ReceiveAssetsCard.tsx:45]: Just curious, why remove these useCallbacks?

**Review by estrattonbailey — APPROVED** [review]

File: src/components/cards/ReceiveAssetsCard.tsx (hunk 3 above)
Comment (benisgold) [reply src/components/cards/ReceiveAssetsCard.tsx:45]: i don't see why having them is beneficial

**Pull request merged**